### PR TITLE
various fixes noticed while merging algomart main into a fork

### PR DESCRIPTION
**Pull request by tj-boyle**

### CODE REVIEW

File: apps/web/src/components/currency-input/currency-input.tsx [hunk 1]
```diff
@@ -56,7 +57,10 @@ export default function CurrencyInput({
     if (onBlur) onBlur(event_)
   }
 
-  const localizedValue = formatCurrency(value, locale, currency, conversionRate)
+  const localizedValue =
```
Comment (tj-boyle) [inline apps/web/src/components/currency-input/currency-input.tsx:60]: This makes it so it only shows the helper localized value if it's not the default currency (USD)

File: apps/api/src/modules/collections/collections.routes.ts [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
   reply: FastifyReply
 ) {
   const collectionsService = request
     .getContainer()
     .get<CollectionsService>(CollectionsService.name)
   // TODO: get language from request
-  const collections = await collectionsService.getAllCollections()
+  const collections = await collectionsService.getAllCollections(
+    request.query.language
+  )
```
Comment (smonn) [inline apps/api/src/modules/collections/collections.routes.ts:15]: I guess that TODO can be removed?

File: apps/api/src/modules/sets/sets.routes.ts [hunk 3]
```diff
@@ -1,14 +1,17 @@
...
 import { SetsService } from '@algomart/shared/services'
 import { FastifyReply, FastifyRequest } from 'fastify'
 
 export async function getSet(
-  request: FastifyRequest<{ Params: Slug }>,
+  request: FastifyRequest<{ Params: Slug; Querystring: Language }>,
   reply: FastifyReply
 ) {
   const setsService = request.getContainer().get<SetsService>(SetsService.name)
   // TODO: get language from request
```
Comment (smonn) [inline apps/api/src/modules/sets/sets.routes.ts:10]: Can remove this TODO, too 😄

**Review by dallashuggins — APPROVED** [review]
💯

File: apps/api/src/modules/sets/sets.routes.ts (hunk 3 above)
Comment (tj-boyle) [reply apps/api/src/modules/sets/sets.routes.ts:10]: Can I, instead, add a TODO to remove the TODO

File: apps/api/src/modules/collections/collections.routes.ts [hunk 4]
```diff
@@ -1,16 +1,18 @@
...
 
 export async function getAllCollections(
-  request: FastifyRequest,
+  request: FastifyRequest<{ Querystring: Language }>,
   reply: FastifyReply
 ) {
   const collectionsService = request
     .getContainer()
     .get<CollectionsService>(CollectionsService.name)
   // TODO: get language from request
```
Comment (smonn) [inline apps/api/src/modules/collections/collections.routes.ts:12]: And this one 😄

**Pull request merged**